substrate/primitives/weights/Cargo.toml

```diff
@@ -35,6 +35,7 @@ std = [
 	"sp-arithmetic/std",
 	"sp-debug-derive/std",
 ]
+runtime-benchmarks = []
```

Comment:
Is there code in `sp-weights` that is guarded by the feature?

Reply:
yes this, we could also deprecate it IMO:
https://github.com/paritytech/polkadot-sdk/blob/18db502172bdf438f086cd5964c646b318b8ad37/substrate/primitives/weights/src/weight_v2.rs#L404-L416

Reply:
If it was never used, let's remove it.